### PR TITLE
Css brochure form

### PR DESCRIPTION
**Pull request by GiancarloMassoni**
Screen Shot 2022-10-12 at 3 23 34 PM

### CODE REVIEW

File: css-brochure-form/index.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    <label for="form">COMPANY EMAIL</label>
+    <input type="text" placeholder="sally@company.co">
+    <label for="form">NAME</label>
+    <input type="text" placeholder="Sally Smith">
+    <label for="form">WHERE ARE YOU HIRING?</label>
+    <input type="text" placeholder="City, Country">
+    <label for="form">HOW MANY ENGINEERS ARE YOU HIRING IN THE NEXT 6 MONTHS?</label>
+    <select name="form" id="selectValue">
+      <option value="">1-2 Engineers</option>
+    </select>
```
Comment (codingcodymiller) [inline css-brochure-form/index.html]: Your labels are not properly configured to pair with the form control element that they are labeling.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by GiancarloMassoni** [comment]
Screen Shot 2022-10-12 at 3 27 56 PM

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Just submitting a quick review so I know when this is updated 👍

**Review by codingcodymiller — APPROVED** [review]
Nice work!

**Pull request merged**